docs/diagnostics/PX1037.md

```diff
@@ -0,0 +1,46 @@
+# PX1037
+This document describes the PX1037 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                 | Type  | Code Fix    | 
+| ------ | ------------------------------------------------- | ----- | ----------- | 
+| PX1037 | Unbound DAC field cannot be used in a key declaration  | Warning (ISV Level 3: Informational) | Available | 
+
+## Diagnostic Description
+The diagnostic checks the fields which are used in a primary, foreign, or unique key declaration. The key declaration cannot contain unbound fields.
+The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
+The diagnostic does not check abstract DACs and DACs which have only unbound DAC properties (fully-unbound DACs used for custom popups and filters for inquiry forms).
+
+To fix the issue, specify a database bound field in a key declaration.
```

Comment:
"...remove unbound fields from the key declaration and replace them with database bound fields"
(there might be multiple database bound fields)